freeipa/src/main/resources/application.yml

```diff
@@ -76,6 +76,7 @@ freeipa:
       enabled: true
       initial-delay-millis: 60000
       fixed-delay-millis: 300000
+      cooldown-millis: 600000
```

Comment:
So why 10minutes? I commented on the doc, but we should get product to tell us what is an acceptable SLA for this. This can be updated later when we have the number from product.

Reply:
changed this to use Duration string (e.g., PT10M). We can update once we determine SLA. In the meantime, this will prevent us from constantly running